x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -870,7 +870,8 @@ export const ModelsList: FC<Props> = ({
     if (pageState.showAll) {
       return items;
     } else {
-      return items.filter((item) => item.supported !== false);
+      // by default show only deployed models or recommended for download
+      return items.filter((item) => item.create_time || item.recommended);
```

Comment:
is the `||` intentional or could this be `??` ?

Reply:
It's safe either way here. Left `||` just in case it's an empty string.